README.md

```diff
@@ -2,6 +2,10 @@
 
 VPC CNI plugins for Amazon ECS and Amazon EKS.
 
+# Pre-requisites for running CNI plugin for EKS Windows
+1. Env variable  `AWS_VPC_CNI_K8S_CONNECTOR_BINARY_PATH` is required to be set. This will be already set in EKS Windows Optimized AMIs.
+Set env variable `AWS_VPC_CNI_K8S_CONNECTOR_BINARY_PATH` to `C:\Program Files\Amazon\EKS\cni\aws-vpc-cni-k8s-connector.exe`.
```

Comment:
Let's set a default value which is `C:\Program Files\Amazon\EKS\cni\aws-vpc-cni-k8s-connector.exe`. If the ENV is set then we can use the same.

Reply:
done